unit_tests/testSchemaParser/testFile.inp

```diff
@@ -0,0 +1,36 @@
+SINGLE_SIK_BLOCK
+  SINGLE_SIK_CARD 5
+
+SINGLE_SRK_BLOCK
+  SINGLE_SRK_CARD 3.0
+
+SINGLE_SBK_BLOCK
+  SINGLE_SBK_CARD true
+
+SINGLE_STR_BLOCK
+  SINGLE_STR_CARD string
+
+
+!Some Comments to ignore
+MULTPL_SBK_BLOCK  MULTPL_SBK_CARD  true MULTPL_SBK_CARD T MULTPL_SBK_BLOCK MULTPL_SBK_CARD False
```

Comment:
oh, so this is all legal syntax?  Nifty.

Reply:
yep.  Just trying to illustrate that it's all keyword based now.  In theory you could have the input be a single line.  Figure this gives users a lot more flexibility for constructing readable inputs.

Reply:
yeah, giving the user some flexibility there is nice.  Long multistate cases, for example, would look nice making use of this.